### PR TITLE
Added API1600 support to artifact bundle

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
Added API1600 support to artifact bundle

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
  - [x] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [x] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Review by chebroluharika — APPROVED** [review]

File: examples/image_streamer/artifact_bundle.rb [hunk 1]
```diff
@@ -20,7 +20,9 @@
 
 i3s_client = {
   url: ENV['I3S_URL'],
-  oneview_client: oneview_client
+  oneview_client: oneview_client,
+  api_version: 1600,
+  ssl_enabled: false
```
Comment (AsisBagga) [inline examples/image_streamer/artifact_bundle.rb]: I am not sure if ssl_enabled should be false by default, can you check for other examples, and follow a similar pattern please?

File: libraries/resource_providers/image_streamer/api300/artifact_bundle_provider.rb [hunk 2]
```diff
@@ -66,6 +66,7 @@ def upload
 
         def extract
           raise "#{@resource_name} #{@name} does not exist in the appliance. Cannot run extract action." unless @item.exists?
+          @item.retrieve! || raise("#{@resource_name} '#{@name}' not found!")
```
Comment (AsisBagga) [inline libraries/resource_providers/image_streamer/api300/artifact_bundle_provider.rb:69]: Not sure whats the use or OR operator here, cant we add "unless" to check if that particular resource's name exist in OV or not and raise exception accordingly?

**Review by AsisBagga — APPROVED** [review]
Apart from the comment mentioned above, i believe PR is good to be merged.

File: examples/image_streamer/artifact_bundle.rb (hunk 1 above)
Comment (VenkateshRavula) [reply examples/image_streamer/artifact_bundle.rb]: done

**Pull request merged**